code/modules/mob/language/station.dm

```diff
@@ -9,6 +9,17 @@
 	flags = RESTRICTED
 	syllables = list("hs","zt","kr","st","sh")
 
+/datum/language/eyepod
+	name = LANGUAGE_EYEPOD
+	desc = "A complex language known instinctively by Eyepods, 'spoken' by emitting modulated radio waves. This version uses high frequency waves for quick communication at short ranges."
+	speech_verb = "babbles"
+	ask_verb = "babbles"
+	exclaim_verb = "babbles"
```

Comment:
Maybe make different babbles. Like babbles questioningly, or babbles excitedly? Just an idea at this point.